src/components/person_info.tsx

```diff
@@ -8,16 +8,16 @@ export default function PersonInfo(props: {
     description: string;
     extraClasses?: string;
 }) {
-    const baseClassNames = `w-[90vw] sm:w-[80vw] md:w-[70vw] shadow-sm whitespace-pre-line shadow-wai-gray bg-pure-white text-xl sm:text-xl md:text-2xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center flex items-center`;
-    const cardClassNames = `${baseClassNames} ${props.extraClasses || ""}`;
-    const tagClassNames = `${props.tagColour ? `!${props.tagColour}` : "text-wai-gray"}`;
+    const baseClassNames = `w-full shadow-sm whitespace-pre-line shadow-wai-gray bg-pure-white text-xl sm:text-xl md:text-2xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-2 md:p-4 justify-self-center`;
+    const cardClassNames = `${baseClassNames} ${props.extraClasses || ""}` + "grid grid-cols-1 md:flex md:flex-row md:items-center";
+    const colour = props.tagColour || "#9372BE";
 
     return (
         <div className={cardClassNames}>
-            <img src={props.picture} className="w-[calc(9vw*7/6)] h-[calc(9vw*7/6)] object-cover" />
+            <img src={props.picture} className="place-self-center w-3/5 min-w-36 max-w-52 md:2/5 aspect-square mb-4 lg:pb-0 lg:w-1/5 object-cover rounded-lg" />
             <div className="flex-1 pl-4 text-left">
                 <div className="text-lg sm:text-xl md:text-2xl mb-2">
-                    {props.name} | <span className={tagClassNames}>{props.tag}</span>
+                    {props.name} | <span className="text-wai-gray" style={{color: colour}}>{props.tag}</span>
```

Comment:
you might think why not use this? https://tailwindcss.com/docs/text-color#arbitrary-values

at build time, tailwind CSS classes are evaluated and unused tags are pruned to reduce bundle sizes sent to the user
- given that the colours come from a CSV that is not read at build time, this is pruned
- inline style tags override the class attributes that just CSS